File: src/my_project/easy_problems/from251to300/find_difference_between_two_arrays.py
from typing import List 

class Solution:
    def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
        
        l1=set()
        l2 = set()
        nums1 = set(nums1)
        nums2 = set(nums2)
        for i in nums1:
            if i not in nums2:
                l1.add(i)
        for i in nums2:
            if i not in nums1:
                l2.add(i)
        return [l1,l2]

File: my_project_typescript/medium_problems_typescript/from51to100/findDifferenceBetweenTwoArrays.ts
function findDifference(nums1: number[], nums2: number[]): number[][] {

    let arrayOne: Array<number> = []
    let arrayTwo: Array<number> = []


    let solution: Array<Array<number>> = []


    for (let a of nums1){
        if (!nums2.includes(a)){
            arrayOne.push(a)
        }
    }

    for (let a of nums2){
        if (!nums1.includes(a)){
            arrayTwo.push(a)
        }
    }

    solution.push(arrayOne)
    solution.push(arrayTwo)

    return solution

};